### PR TITLE
8 deploy safes gelato

### LINKED ISSUE #8: Deploy Safes with Gelato Relay
- [x] Stop using the `@safe-global/protocol-kit` library for deploying Safes and just use Gelato for everything
- [x] Remove the `/safes` endpoint
- [x] Fix tests and docs

### PR DESCRIPTION
**Pull request by juanenrisley**
Closes #8

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -2,12 +2,5 @@ NODE_ENV=development
...
-GELATO_RELAY_API_KEY=
 RPC_URL=http://127.0.0.1:8545
-CHAIN_ID=1337
-FASTIFY_LOG_LEVEL=info
-SAFE_MASTER_COPY_ADDRESS=0x2612Af3A521c2df9EAF28422Ca335b04AdF3ac66
-SAFE_PROXY_FACTORY_ADDRESS=0x9b1f7F645351AF3631a656421eD2e40f2802E6c0
-FALLBACK_HANDLER_ADDRESS=0x67B5656d60a809915323Bf2C40A8bEF15A152e3e
-MULTI_SENT_ADDRESS=0xe982E462b094850F12AF94d21D470e21bE9D0E9C
-MULTI_SEND_CALL_ONLY_ADDRESS=0x0290FB167208Af455bB137780163b7B7a9a10C16
+FASTIFY_LOG_LEVEL=info
```
Comment (JacqueGM) [inline .env.example:6]: EOF

File: __tests__/transactions.js [hunk 2]
```diff
@@ -27,6 +24,29 @@ describe('Transactions features:', () => {
...
+              data: '0x6a761202000000000000000000000000cfeb869f69431e42cdb54a4f4f105c19c080a6010000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000014000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000001c000000000000000000000000000000000000000000000000000000000000000449951d62f00000000000000000000000090f8bf6a479f320ead074411a4b0e7944ea8c9c1000000000000000000000000000000000000000000000000000000000000000a000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000041eb667a968b979d8406df46c7fedbc2cd6e9046a10f379baa27ddecf560b233261aba9bdc6aa8f391e45156bdac5165f12b1c1ceae2729353ebf12810302dacbc1c00000000000000000000000000000000000000000000000000000000000000',
```
Comment (JacqueGM) [inline __tests__/transactions.js:37]: is it the same data for three different transactions?

File: .env.example (hunk 1 above)
Comment (juanenrisley) [reply .env.example]: Added

File: package.json [hunk 3]
```diff
@@ -41,4 +40,4 @@
     "lint-staged": "13.2.2",
     "prettier": "2.8.8"
   }
-}
+}
```
Comment (JacqueGM) [inline package.json:43]: EOF

Comment (juanenrisley) [reply package.json]: done

File: __tests__/transactions.js (hunk 2 above)
Comment (juanenrisley) [reply __tests__/transactions.js:37]: Yes, it is

**Review by JacqueGM — APPROVED** [review]
LGTM ^^

**Pull request merged**